plex_trakt_sync/trakt_list_util.py

```diff
@@ -5,17 +5,45 @@
 from trakt.movies import Movie
 from trakt.tv import TVEpisode
 from trakt.users import UserList
+from trakt.core import get
+from trakt.utils import slugify, extract_ids
 
 from plex_trakt_sync.logging import logger
 from plex_trakt_sync.plex_api import PlexApi
 
 
+class LazyUserList(UserList):
+    @get
+    def get_items(self):
+        data = yield 'users/{user}/lists/{id}/items'.format(
+            user=slugify(self.creator), id=self.slug)
+        for item in data:
+            if 'type' not in item:
+                continue
+            item_type = item['type']
+            item_data = item.pop(item_type)
+            extract_ids(item_data)
+            self._items.append((item_type + 's', item_data['trakt']))
+        yield self._items
+
+    @classmethod
+    @get
+    def _get(cls, title, creator):
```

Comment:
`_get` override should not be needed after https://github.com/glensc/python-pytrakt/pull/3